packages/tools/src/stateManagement/annotation/annotationState.ts

```diff
@@ -84,15 +86,17 @@ function addAnnotation(
 
   manager.addAnnotation(annotation, groupKey);
 
-  // if the annotation manager selector is an element, trigger the
-  // annotation added event for that element.
-  if (annotationGroupSelector instanceof HTMLDivElement) {
-    triggerAnnotationAddedForElement(annotation, annotationGroupSelector);
-  } else {
-    // if no element is provided, render all viewports that have the
-    // same frame of reference.
-    // Todo: we should do something else here for other types of annotation managers.
-    triggerAnnotationAddedForFOR(annotation);
+  if (!suppressEvents) {
```

Comment:
I'm really not keen on suppressing events here - it means that another viewport depending on the events that happens to be displaying a slice which would get included will not end up being displayed.  As well, OHIF and other viewers based on CS3D may need the events to maintain their internal state.

Reply:
It would be better to use an extra flag that someone can check to see that these events are intermediate or interpolated events - you might add another partial set of values instead of the suppress events flag, and within that include the status of what is happening on the interpolation events.  Then, anyone listening who knows nothing about them can just act as normal, and any place where you need to not do the default handling, you can just check the extra flags.